packages/block-editor/src/store/selectors.js

```diff
@@ -2909,16 +2909,6 @@ export function isBlockVisible( state, clientId ) {
 	return state.blockVisibility?.[ clientId ] ?? true;
 }
 
-/**
- * Returns the currently hovered block.
- *
- * @param {Object} state Global application state.
- * @return {Object} Client Id of the hovered block.
- */
-export function getHoveredBlockClientId( state ) {
-	return state.hoveredBlockClientId;
-}
```

Comment:
I'm in two minds about removing the action/selectors completely. Not sure if an option might be to have it return `undefined` and log a deprecation message. It at least prevents the editor from blowing up completely.

The action could do the same (not dispatch anything), though I'm not sure if an action returning `undefined` is an issue - would it be ignored?

Reply:
I was also hesitant about removing these vs deprecation. But here's why I decided to do so:

* The `https://wpdirectory.net/` returns no results for them.
* These actions/selectors shouldn't have been made public in the first case.
* The revert should've happened in https://github.com/WordPress/gutenberg/pull/65759, but that ship has sailed :(

Reply:
@talldan, added changelog entry and marked this a breaking change.

Reply:
> The https://wpdirectory.net/ returns no results for them.

I don't think this is completely exhaustive, which is why I'm hesitant about it. There's also plugins outside the repo and custom code added to individual sites that it doesn't cover. 

I agree though, I can't imagine anyone using the APIs.

Alternative compromise could be to keep the functions with deprecations for a few releases, then follow up and remove them after a couple of plugin releases with the deprecation message active.